config/generate/kinto.tpl

```diff
@@ -53,22 +53,22 @@ multiauth.policies = basicauth
 # http://kinto.readthedocs.org/en/latest/configuration/settings.html#client-caching
 #
 # Every bucket objects objects and list
-# kinto.bucket_cache_expires_seconds = 3600
+ kinto.bucket_cache_expires_seconds = 3600
 #
 # Every collection objects and list of every buckets
-# kinto.collection_cache_expires_seconds = 3600
+ kinto.collection_cache_expires_seconds = 3600
 #
 # Every group objects and list of every buckets
-# kinto.group_cache_expires_seconds = 3600
+ kinto.group_cache_expires_seconds = 3600
 #
 # Every records objects and list of every collections
-# kinto.record_cache_expires_seconds = 3600
+ kinto.record_cache_expires_seconds = 3600
 #
 # Records in a specific bucket
-# kinto.blog_record_cache_expires_seconds = 3600
+ kinto.blog_record_cache_expires_seconds = 3600
 #
 # Records in a specific collection in a specific bucket
-# kinto.blog_article_record_cache_expires_seconds = 3600
```

Comment:
You can let these commented, they are just examples of values that could be turned on.